addons/web/static/src/js/libs/jquery.js

```diff
@@ -131,6 +131,10 @@ $.fn.extend({
             if ($el.isScrollable()) {
                 return $el;
             }
+            // Avoid infinite loop in case of erroneous call.
+            if ($el.length === 0) {
+                return false;
```

Comment:
When I said that a jquery method can return false, I meant for something like `$().isScrollable()` -> false because no element. But closestScrollable should return an empty jQuery element (and in any case, you should have updated the docstring and the docstring of all callers).